README.md

```diff
@@ -1,62 +1,84 @@
 
 #### Usage instruction
 ```
-docker run -v ${PWD}:/wd genomenexus/gn-annotation-pipeline:master --filename /wd/input.txt  --output-filename /wd/output.txt
+docker pull genomenexus/gn-annotation-pipeline:master 
+```
+```
+docker run -v ${PWD}:/wd genomenexus/gn-annotation-pipeline:master java -jar annotationPipeline.jar --filename /wd/input.txt  --output-filename /wd/output.txt
 ```
 - `-v ${PWD}:/wd`: This option maps the current working directory to a volume within the Docker container at the path `/wd`. This makes it possible for files in the host directory to be accessed and modified between container and host.
 - `--filename /wd/input.txt`: This option specifies the input file location at `/wd/input.txt`, which should be under the same directory as output file.
-
 - `--output-filename /wd/output.txt`: This option specifies the output file where the annotated results will be saved. The file will be created at `/wd/output.txt`, which should be under the same directory as input file.
 
-#### View logging file
-To enable logging in the Genome Nexus Annotation Pipeline and access the log file, you need to mount your local path to view the log file locally. By default, the log file is stored at `/genome-nexus-annotation-pipeline/logs/genome-nexus-annotation-pipeline.log`. Use the following command as an example:
+### Logging
+
+> ![IMPORTANT]
+> Logging via docker has been changed to `stdout` by default.
+
+##### View logging file (if built with file logging config)
+To access the log file, you need to mount your local path to view the log file locally. 
+
+By default, the log file is stored at 
+`/genome-nexus-annotation-pipeline/logs/genome-nexus-annotation-pipeline.log`. 
+
+Use the following command as an example
 ```
-docker run  -v ${PWD}:/wd -v ${PWD}/logs:/genome-nexus-annotation-pipeline/logs genomenexus/gn-annotation-pipeline:master --filename  /wd/input.txt  --output-filename /wd/output.txt
+docker run -v ${PWD}:/wd -v ${PWD}/logs:/genome-nexus-annotation-pipeline/logs genomenexus/gn-annotation-pipeline:master java -jar annotationPipeline.jar --filename /wd/input.txt --output-filename /wd/output.txt
 ```
-- `-v ${PWD}/logs:/genome-nexus-annotation-pipeline/logs` flag mounts the logs directory inside the current working directory to the corresponding directory inside the Docker container.
-- Other flags are the same as above
+- `-v ${PWD}/logs:/genome-nexus-annotation-pipeline/logs` flag mounts the current working directory to the corresponding log directory inside the Docker container.
 
-Make sure to adjust the file paths according to your specific requirements. Once the command is executed, the log file will be generated and stored in the logs directory within your local directory. 
+> ![NOTE]
+> Make sure to adjust the file paths according to your specific requirements. 
+> The log file will be generated and stored in the logs directory within your local directory. 
```

Comment:
@hweej is this still necessary? Wondering if we can delete it now that it goes to stdout in Docker